integrations/slack/oauth.go

```diff
@@ -11,17 +11,20 @@ import (
 	"go.autokitteh.dev/autokitteh/integrations/slack/api/bots"
 	"go.autokitteh.dev/autokitteh/integrations/slack/internal/vars"
 	"go.autokitteh.dev/autokitteh/sdk/sdkintegrations"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
 // handler is an autokitteh webhook which implements [http.Handler]
 // to receive and dispatch asynchronous event notifications.
 type handler struct {
 	logger *zap.Logger
+	oauth  sdkservices.OAuth
+	vars   sdkservices.Vars
 }
 
-func NewHandler(l *zap.Logger) http.Handler {
-	return handler{logger: l}
+func NewHandler(l *zap.Logger, o sdkservices.OAuth, v sdkservices.Vars) handler {
```

Comment:
I'm assuming the actual usage of the OAuth service is pending in this PR and not a separate one? Otherwise, please postpone this addition.

Reply:
I'm not sure. I think this is a relic of when I was trying something different. I removed it.